_data/tabs.yml

```diff
@@ -1,17 +1,20 @@
 - name: Home
   link: /
-  
+
 - name: Announcements
   link: /announcements/
-  
+
 - name: Talks
   link: /talks/
-  
-- name: About
-  link: /about.html
-  
+
+- name: Projects
+  link: /projects/
+
+#- name: About
+#  link: /about/
+
 - name: Get Involved
-  link: /get-involved.html
+  link: /get-involved/
 
-- name: CampusGroups
-  link: https://campusgroups.rit.edu/student_community?club_id=16071
```

Comment:
I think it is best to keep CampusGroups in the footer or clearly linked from the website. Even if we are not actively using it, it is still the platform we use for critical club functions, like elections. It will be helpful to keep this visible and present on our website.

Reply:
It's still on both the footer ("RIT Club Page") and on the "get involved" page. I don't want it on the top navbar because it takes up a decent amount of space and we haven't really done much with it. Do you have any suggestions?

Reply:
It's still on both the footer ("RIT Club Page") and on the "get involved" page. I don't want it on the top navbar because it takes up a decent amount of space and we haven't really done much with it. Do you have any suggestions?

Reply:
Oh, I missed it in the footer. Instead, could you make the footer and "get involved" page more explicit as "CampusGroups" instead of "RIT Club Page"?

Reply:
Fixed